qtpy/tests/test_qtquick.py

```diff
@@ -37,9 +39,9 @@ def test_qtquick():
     assert QtQuick.QSGOpacityNode is not None
     if not PYSIDE2:
         assert QtQuick.QSGOpaqueTextureMaterial is not None
-    #assert QtQuick.QSGRectangleNode is not None
-    #assert QtQuick.QSGRenderNode is not None
-    #assert QtQuick.QSGRendererInterface is not None
+    # assert QtQuick.QSGRectangleNode is not None
+    # assert QtQuick.QSGRenderNode is not None
+    # assert QtQuick.QSGRendererInterface is not None
```

Comment:
I think something similar happens with these modules (available on PyQt5 but not on PySide2). Could you uncomment the asserts and move them to/add to them the missing validation?